tests/test_viz.py

```diff
@@ -1,35 +1,34 @@
+import pytest
 import numpy as np
-from pyriemann.utils.viz import (plot_confusion_matrix, plot_embedding,
-                                 plot_cospectra)
+from conftest import covmats
 
+try:
+    from pyriemann.utils.viz import (
+        plot_confusion_matrix,
+        plot_embedding,
+        plot_cospectra,
+    )
```

Comment:
this is not the right fix for me.

You should import plt inside the plot functions. Not at the top of the file at:

https://github.com/pyRiemann/pyRiemann/blob/master/pyriemann/utils/viz.py#L8

then you should decorate each test with something that skips the test if matplotlib is not installed

Reply:
Ok, I have done it like that for stats.py. For the viz.py, I was wondering if it makes sense as all function are for generating plots.